config-provider-library/src/main/java/com/aevi/sdk/pos/flow/config/ui/FpsSettingsFragment.java

```diff
@@ -0,0 +1,205 @@
+import com.aevi.ui.library.view.settings.SettingControl;
+import com.aevi.ui.library.view.settings.SettingIntegerInput;
+import com.aevi.ui.library.view.settings.SettingSwitch;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import io.reactivex.functions.Consumer;
+
+@SuppressWarnings("ResultOfMethodCallIgnored")
+public class FpsSettingsFragment extends BaseFragment {
+
+    private static final int TIMEOUT_MAX = 3600;
+    private static final int ROW_LIMIT_MAX = 10000;
+
+    @BindView(R2.id.split_timeout)
+    SettingIntegerInput splitTimeout;
+
+    @BindView(R2.id.flow_response_timeout)
+    SettingIntegerInput flowResponseTimeout;
+
+    @BindView(R2.id.payment_response_timeout)
+    SettingIntegerInput paymentResponseTimeout;
+
+    @BindView(R2.id.select_timeout)
+    SettingIntegerInput selectTimeout;
+
+    @BindView(R2.id.status_update_timeout)
+    SettingIntegerInput statusUpdateTimeout;
+
+    @BindView(R2.id.database_row_limit)
+    SettingIntegerInput databaseRowLimit;
+
+    @BindView(R2.id.abort_on_flow_error)
+    SettingSwitch abortOnFlow;
+
+    @BindView(R2.id.abort_on_payment_error)
+    SettingSwitch abortOnPayment;
+
+    @BindView(R2.id.allow_status_bar_access)
+    SettingSwitch allowStatusBar;
+
+    @BindView(R2.id.enable_legacy_pa_support)
+    SettingSwitch enableLegacySupport;
+
+    @BindView(R2.id.multi_device_support)
+    SettingSwitch multiDeviceSupport;
+
+    @BindView(R2.id.currency_conversion_support)
+    SettingSwitch currencyConversionSupport;
+
+    @BindView(R2.id.filter_flow_services_by_type)
+    SettingSwitch filterFlowServices;
+
+    @BindView(R2.id.always_call_preflow)
+    SettingSwitch alwaysCallPreFlow;
+
+    @Inject
+    SettingsProvider settingsProvider;
+
+    @Inject
+    Context appContext;
+
+    @Inject
+    ProviderAppDatabase appDatabase;
+
+    @Inject
+    ProviderAppScanner appScanner;
+
+    private FpsSettings fpsSettings;
+    private boolean hasChanges;
+    private boolean readOnlyMode;
+
+    public FpsSettingsFragment() {
+    }
+
+    public FpsSettingsFragment(boolean readOnlyMode) {
+        this.readOnlyMode = readOnlyMode;
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ConfigComponentProvider.getFpsConfigComponent().inject(this);
+        setRetainInstance(true);
+        fpsSettings = settingsProvider.getFpsSettings();
+    }
+
+    @Override
+    public int getLayoutResource() {
+        return R.layout.fragment_fps_settings;
+    }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        notifyChangeIfNecessary();
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        setupNumericInput();
+        setupSwitches();
+        if (readOnlyMode) {
+            setReadOnly(splitTimeout, paymentResponseTimeout, flowResponseTimeout, selectTimeout, abortOnFlow, abortOnPayment,
```

Comment:
Minor - You could use Butterknife ViewList and Action.Setter here would make this more readable but might actually be more verbose/boilerplate.